nerfstudio/models/splatfacto.py

```diff
@@ -686,6 +700,23 @@ def _get_background_color(self):
             raise ValueError(f"Unknown background color {self.config.background_color}")
         return background
 
+    def _apply_bilateral_grid(self, rgb: torch.Tensor, cam_idx: int, H: int, W: int) -> torch.Tensor:
+        # make xy grid
+        grid_y, grid_x = torch.meshgrid(
```

Comment:
would be faster to create this meshgrid on the gpu and avoid the `.to(self.device)` line just below this